scripts/corelli/calibration/database.py

```diff
@@ -36,64 +37,165 @@ def has_valid_columns(table:TableWorkspace)->bool:
     return True
 
 
-def filename_bank_table( bankID:int, database_path:str, table_type:str = 'calibration' ) -> str:
+def filename_bank_table( bankID:int, database_path:str, date:str, table_type:str = 'calibration' ) -> str:
     """
     Function that returns the absolute path for a bank file using corelli format:
     database_path/bank0ID/type_corelli_bank0ID_YYYYMMDD.nxs
     :param bankID bank number that is calibrated
+    :param date format YYYYMMDD
     :param database_path location of the corelli database (absolute or relative)
            Example: database/corelli/ for
                     database/corelli/bank001/
                     database/corelli/bank002/
     :param type 'calibration', 'mask' or 'acceptance'
     """
+
+    verify_date_YYYYMMDD('filename_bank_table', date)
     subdirectory:str = database_path + '/' + 'bank' + str(bankID).zfill(3)
```

Comment:
Let me verify the ordering after being sorted and I might have to add extra logic, since string-wise `bank10 < bank9`

Reply:
As we discussed offline, this is just preferred so `ls` will show directories and files in `alpha-numeric` order.  Thanks!